core/src/ballot_box.rs

```diff
@@ -0,0 +1,378 @@
+use bytemuck::{Pod, Zeroable};
+use jito_bytemuck::{
+    types::{PodU128, PodU64},
+    AccountDeserialize, Discriminator,
+};
+use shank::{ShankAccount, ShankType};
+use solana_program::{account_info::AccountInfo, msg, program_error::ProgramError, pubkey::Pubkey};
+use spl_math::precise_number::PreciseNumber;
+
+use crate::{constants::PRECISE_CONSENSUS, discriminators::Discriminators, error::TipRouterError};
+
+#[derive(Debug, Clone, PartialEq, Eq, Copy, Zeroable, ShankType, Pod, ShankType)]
+#[repr(C)]
+pub struct Ballot {
+    merkle_root: [u8; 32],
+    max_total_claim: PodU64,
+    max_node_count: PodU64,
```

Comment:
We may not need these two fields, just the merkle root itself. (the max total claim and max node count here would be different than what gets passed into the Tip Distribution program instruction, since those are unique per vote account)

Reply:
10-4, I'll keep the reserved bytes just in case